source/portable/os/ota_os_freertos.c

```diff
@@ -279,8 +294,12 @@ OtaErr_t OtaStartTimer_FreeRTOS( OtaTimerId_t otaTimerId,
 
 OtaErr_t OtaStopTimer_FreeRTOS( OtaTimerId_t otaTimerId )
 {
+    configASSERT( ( otaTimerId >= OtaRequestTimer ) && ( otaTimerId < OtaNumOfTimers ) );
```

Comment:
Same as above i.e.

```c
OtaErr_t otaErrRet = OTA_ERR_UNINITIALIZED;
BaseType_t retVal = pdFALSE;

configASSERT( ( otaTimerId >= OtaRequestTimer ) && ( otaTimerId < OtaNumOfTimers ) );
```

Reply:
Addressed in https://github.com/aws/ota-for-aws-iot-embedded-sdk/commit/5b69254cbead6a8843cb66190090e39e9940e6ce